extensions/wikia/LookupContribs/LookupContribs.php

```diff
@@ -1,8 +1,71 @@
 <?php
-define ( 'LOOKUPCONTRIBS_HELP', 'This special page can be used to display user\'s contributions on multiple wikias. For displaying the list, give username below.' ) ;
-define ( 'LOOKUPCONTRIBS_PATH', '/' ) ;
-/* cache? no cache? */
-define ( 'LOOKUPCONTRIBS_NO_CACHE', 'false' ) ;
```

Comment:
This double negative LOOKUPCONTRIBS_NO_CACHE doesn't really need to be here.  We should always cache.  If there is a problem with caching, we should fix it.